config/config.go

```diff
@@ -50,7 +49,6 @@ type Config struct {
 	MetricsWALPath       string
 	LogLevel             string // one of "ERROR", "WARN", "INFO", "DEBUG", "TRACE"
 	LogPath              string
-	LogPrefix            string
```

Comment:
Will mention the idea behind this feature. The prefix was introduced mostly for testing/adhoc use cases. E.g. I had a vm which could run multiple instances of host-metering, e.g. logging to a different file but also each having a different prefix, making it possible to add datetime (which is by default in journal, but not in file, when logging to file). + it was possible to add custom string to be able to distinguish the host-metering instance simply from the line. 

Not sure if this is still need. + Logrus might solve the same differently.

Reply:
Hm. As far as I understand the CustomFormatter, which includes the datetime, should be used in either case for stderr and file logging but I might be wrong about this. @ConorC117 can you validate if this is the case? 
Once you set LogPath to a value != "" you should be able to test this.

Reply:
Correct me if I'm wrong, but I think the decision is made in logger.go 

https://github.com/RedHatInsights/host-metering/blob/ac4f67bb15593c1201eed653e92eac03d83ef3c7/logger/logger.go#L98-L101

Reply:
Yes that's the decision point. By default the logger printed to stderr which means to journal when it is running as a service.

Reply:
Oh thanks for pointing me there :)
But then its still not clear to me whats the relation between filename and the log line prefiix. 😅 ? Did the prefix serve as a reference to the chosen filename? If thats the case, we could provide a default key value pair of `filename=something.log` in the logging fields. Does that make sense? :)

Reply:
@FKolwa Just tested the CustomFormatter and can confirm that's the case! The logs contain the datetime for both file logging and stderr

Reply:
@ConorC117 Thanks for validating, I like that the logging behavior stays consistent across different output formats.

@miyunari Yes, that's one of the things it can do. Although, there is no explicit relation between filename and LogPrefix unless you decided to add it using one of the available flags:
https://github.com/RedHatInsights/host-metering/blob/ac4f67bb15593c1201eed653e92eac03d83ef3c7/contrib/man/host-metering.conf.5#L110-L148
The prefix currently gives you the ability to customize loglines by adding any one of these:
- some string of text you pass in
- the date in a format you like (was only really necessary for file logging)
- the filename

The question is, what information do we want to have in each logline? If I understand @pvoborni correctly this was mainly used for dev-testing, so the choice is on us. We don't need the prefix anymore for adding the date, so that's great. I also don't necessarily need the ability to output the log filename.

But, from a developers perspective, I still want to have some way of distinguishing between multiple running host-metering instances during development. That said, I like your idea @miyunari to add a field with an instance ID. Anyways, we should take this discussion offline and add missing features in a follow-up task. @ConorC117 @miyunari What do you think?

Reply:
@FKolwa That makes a lot of sense to me! I'll go ahead and make the requested changes, update this PR, then create a new task for distinguishing between multiple running host-metering instances. I also like your idea @miyunari of adding a field with the instance ID so I will aim for that approach too!